React/Modules/RCTDevMenu.m

```diff
@@ -185,7 +185,7 @@ - (instancetype)init
       [weakSelf.bridge.eventDispatcher sendDeviceEventWithName:@"toggleElementInspector" body:nil];
     }]];
 
-    _webSocketExecutorName = [_defaults objectForKey:@"websocket-executor-name"] ?: @"Chrome";
+    _webSocketExecutorName = [_defaults objectForKey:@"websocket-executor-name"] ?: @"JS Remotely";
```

Comment:
We want to keep the word "remote" in the UI somewhere because that's one of the key differences between using JSC and one of these remote JS VMs... especially if we start experimenting with JSVMs that run on the UI thread and need to be on the device.